tests/playwright/helpers/misc.ts

```diff
@@ -0,0 +1,48 @@
+import config from '@pw-config'
```

Comment:
Could you move `tests/playwright/tsconfig.json` here as well? Because `@pw-config` is an alias defined in the `tsconfig.json` file.

Reply:
My mistake, it's already here.